crates/bevy_asset/src/io/android_asset_io.rs

```diff
@@ -32,7 +35,10 @@ impl AndroidAssetIo {
 impl AssetIo for AndroidAssetIo {
     fn load_path<'a>(&'a self, path: &'a Path) -> BoxedFuture<'a, Result<Vec<u8>, AssetIoError>> {
         Box::pin(async move {
-            let asset_manager = ndk_glue::native_activity().asset_manager();
+            let asset_manager = bevy_android::ANDROID_APP
```

Comment:
Yeah, I do think it would be nicer to try and avoid having this be turned into global/static state if possible.